interface/batchcom/batchcom.php

```diff
@@ -24,280 +29,217 @@
+                    <input type="text" class="datepicker" name="app_s" placeholder="any date">
+                    <div class="text-center"><?php xl('to','e'); ?></div>
+                    <input type="text" class="datepicker" name="app_e" placeholder="any date">
+            </div>
+            <!-- later gator    <br>Insurance: <SELECT multiple NAME="insurance" Rows="10" cols="20"></SELECT> -->
+            <div class="col-md-3 well">
+                <select name="and_or_seen_within">
+                    <option value="AND"><?php xl('And','e')?></option>
+                    <option value="OR"><?php xl('Or','e')?></option>
+                    <option value="AND NOT"><?php xl('And not','e')?></option>
+                    <option value="OR NOT"><?php xl('Or not','e')?></option>
+                </select>
+                <label for="app_s"><?php xl('Seen within','e')?>:</label>
+                    <input type="text" class="datepicker" name="seen_since" placeholder="any date">
+                    <div class="text-center"><?php xl('to','e'); ?></div>
+                    <input type="text" class="datepicker" name="seen_before" placeholder="any date">
+            </div>
+        </div>
+        <div class="email row">
+            <div class="col-md-6 col-md-offset-3 well">
+                <div class="col-md-6">
+                    <label for="email_sender"><?php xl('Email Sender','e'); ?>:</label>
+                    <input type="text" name="email_sender" placeholder="your@email.email">
+                </div>
+                
+                <div class="col-md-6">
+                    <label for="email_subject"><?php xl('Email Subject','e'); ?>:</label>
+                    <input type="text" name="email_subject" placeholder="From your clinic">
+                </div>
+                <div class="col-md-12">
+                    <label for="email_subject"><?php echo xlt('Email Text, Usable Tag: ***NAME*** , i.e. Dear ***NAME***{{Do Not translate the ***NAME*** elements of this constant.}}'); ?>:</label>
+                </div>
+                <div class="col-md-12">
+                    <textarea name="email_body" id="" cols="40" rows="8"></textarea>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-4 col-md-offset-4 text-center">
+                <input type="hidden" name="form_action" value="process">
+                <input type="submit" name="submit" class="btn btn-primary" value="<?php xl("Process (can take some time)",'e'); ?>">
+            </div>
+        </div>
+    </form>
+</main>
+</body>
+
+<script>
+    (function() {
+        var email = document.querySelector('.email');
+        var process = document.querySelector('select[name="process_type"]');
+        function hideEmail() {
+            if (process.value !== '<?php echo $process_choices[1]; ?>') { email.style.display = 'none'; } else { email.style.display = ''; }
```

Comment:
I think the  'form-control' in the datepicker input should also take care of the width issue